backend/role.py

```diff
@@ -0,0 +1,11 @@
+from typing import List
+
+
+class Role:
```

Comment:
I think there is no reason to use class there. Define roles using `enum` and then implement all functions that belongs to `roles` entity there